ctyle.css

```
@@ -133,3 +140,283 @@ img {
     gap: 20px;
   }
 }
+
+/* about */
+.about {
+  background-color: var(--color-background-black);
+  padding: 80px 0px;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px 40px;
+  gap: 40px;
+}
+
+.about-main-copy {
+  margin: 0;
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  line-height: 48.17px;
+  letter-spacing: 0.02em;
+  color: var(--color-white);
+}
+
+.about-main-copy-accent1 {
+  color: var(--color-accent-red);
+}
+
+.about-main-copy-accent2 {
+  color: var(--color-accent-yellow);
+}
+
+.about-section-body {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 800px;
+  gap: 20px;
+}
+
+.about-section-body-summary {
+  color: var(--color-white);
+}
+
+.about-section-body-logo {
+  height: 180px;
+}
+
+@media (max-width: 750px) {
+  .about {
+    padding: 0px 0px;
+  }
+
+  .about-container {
+    padding: 40px 20px;
+  }
+
+  .about-main-copy {
+    font-size: 24px;
+    line-height: 36.13px;
+    letter-spacing: 0em;
+  }
+
+  .about-section-body {
+    width: auto;
+  }
+
+  .about-section-body-logo {
+    display: none;
+  }
+}
+
+/* courses */
+.courses {
+  background-color: var(--color-background-navy);
+  padding: 80px 0px;
+}
+
+.courses-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px 40px;
+  gap: 40px;
+}
+
+.courses-main-copy {
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  line-height: 48.17px;
+  letter-spacing: 0.02em;
+  color: var(--color-white);
+}
+
+.courses-card {
+  display: flex;
+  flex-direction: column;
+  width: 800px;
+  gap: 20px;
+}
+
+.courses-card-frontman,
+.courses-card-backing,
+.courses-card-lead,
+.courses-card-solo {
```

```suggestion
.courses-card {
```
このあたりはクラスを共通化させてしまっても良いと思います。
それぞれに対して別のスタイルを当てたくなったタイミングで別のクラスを設定していきましょう。

理由としては、クラスを設定するのは必要最小限に留めるのがベストプラクティス（[YAGNI原則](https://wa3.i-3-i.info/word17076.html)）とされているからです。
そのため、コードを読む側としては4つのクラスが設定されていると、当然それぞれに別々のスタイルが当たっているものと考え、コードリーディングの負担が上がってしまいます。

このあたりは雰囲気の部分があり、慣れるまで難しいと思うのですが、コードを書いていくうちに徐々に分かるようになってくるかと思います！！